python/taichi/lang/__init__.py

```diff
@@ -1007,11 +1008,11 @@ def is_arch_supported(arch):
         return False
 
 
-def supported_archs():
-    """Gets all supported archs on the machine.
+def expected_archs():
```

Comment:
Can we move this function from init to testing.py? Do we still want to expose it to users?